packages/@aws-cdk/aws-ecs/lib/service.ts

```diff
@@ -0,0 +1,92 @@
+import cdk = require('@aws-cdk/cdk');
+import { ClusterName } from './cluster';
+import { cloudformation } from './ecs.generated';
+import { TaskDefinitionArn } from './task-definition';
+
+export interface ServiceProps {
+    /**
+     * Cluster where service will be deployed
+     */
+    cluster: ClusterName; // should be required? do we assume 'default' exists?
```

Comment:
I would say we take `Cluster` as an argument (not `ClusterName`). We can always represent the "default" cluster as a special singleton object.